rdflib/parser.py

```diff
@@ -47,9 +47,43 @@
     "URLInputSource",
     "FileInputSource",
     "PythonInputSource",
+    "iri2uri",
 ]
 
 
+def iri2uri(iri: str) -> str:
```

Comment:
I'm a bit ambivalent to having this as a public function, it may be useful but it is not really that important for working with RDF and in this case it only fixes an issue which is not specific to RDF but more specific to network interaction.

Also I would somewhat favor having this in `rdflib.utils` instead of `rdflib.graph` - as I think we will likely end up using it in JSON-LD and SPARQL code in future.

So my suggestion would be `rdflib.util._iri2uri` instead of `rdflib.graph.iri2uri`.

Reply:
Happy to make the change on request.

Reply:
Done now.